python-library/test_signer_burn_transaction.py

```diff
@@ -0,0 +1,81 @@
+import asyncio
+import subprocess
+import sys
+from fullservice import FullServiceAPIv2 as v2
+import pytest
+import json
+import os
+
+repo_root_dir = (
+    subprocess.check_output("git rev-parse --show-toplevel", shell=True)
+    .decode("utf8")
+    .strip()
+)
+sys.path.append(
+    "{}/.internal-ci/test/fs-integration".format(repo_root_dir)
+)  # we're importing the basic.py file as the integration test framework
+
+from basic import TestUtils as Utils
+
+fs = v2()
+
+
+@pytest.mark.asyncio
+async def test_burn_transaction(amount_pmob: int = 600000000):
+    utils = Utils()
+    Utils.get_mnemonics()
+    alice = await utils.init_test_accounts(0, "alice", True)
+    alice_export = await fs.export_account_secrets(alice.id)
+    entropy = (
+        alice_export.get("result")
+        .get("account_secrets")
+        .get("mnemonic")
+        .removeprefix("(")
+        .removesuffix(")")
+    )  # clean up entropy response
+
+    os.system(f"../target/release/transaction-signer create -n alice {entropy}")
+
+    balance_before = int(
+        (await fs.get_account_status(alice.id))
+        .get("result")
+        .get("balance_per_token")
+        .get("0")
+        .get("unspent")
+    )
```

Comment:
A helper function for this would be nice
```suggestion
    balance_before = get_unspent_for_account(alice.id)
```